### ISSUE TITLE
Visualize Impact of E2E Tests on VFS Engineer Experience

### ISSUE THREAD

**Issue by pjhill** [department-of-veterans-affairs/va.gov-team#44574]
# Description
Given the data available regarding VFS engineer sentiment about E2E testing visualize the following.

# Tasks
- [x] Create a dashboard for platform-wide visuals of VFS engineer sentiment about E2E testing
- [x] ~Visualize the number of hours VFS engineers have spent blocked by failing e2e tests~ WE DISCUSSED THIS IN DETAIL AND DECIDED NOT TO PURSUE THIS METRIC BECAUSE THE RESULT WOULD ONLY BE A GUESS. INSTEAD WE'RE TRACKING E2E FAILURES AGAINST MAIN.
- [x] Visualize the time it takes for a CI workflow to complete. This indicates how log it takes an engineer to get the results of the latest change they wish to integrate.
- [x] Visualize VFS engineer sentiment regarding continuous integration workflows and e2e testing over time

# Acceptance Criteria
- [x] A dashboard containing VFS engineer sentiment about CI and E2E tests for the entire platform is publicly available

**Comment by holdenhinkle** [comment]
Here's the VFS survey data regarding Platform for Q1 and Q2:

- 2022 - https://app.mural.co/t/adhocvetsgov9623/m/adhocvetsgov9623/1645655432415/26f1766bc490b410bff347eb2d97e4904d090f79?sender=302066bc-6019-4142-9bc3-71ea21b81f91
- q1 - https://app.mural.co/t/adhocvetsgov9623/m/adhocvetsgov9623/1645571236979/08f59565966ee0e213128c95d374a3d9baeb78e6?sender=c57ce896-b410-4377-94a4-87aec65e1a5a
- q2 - https://app.mural.co/t/adhocvetsgov9623/m/adhocvetsgov9623/1654536055645/667377508f107d1b1c830167f2807d90d6c0eb18?sender=0e3bbfd5-939f-4e3f-b660-8e08491ce440

I searched the documents for the following keywords and couldn't find anything about CI, E2E tests, Cypress tests, etc.:

- test
- e2e
- cypress
- github
- actions

There's no VFS engineer sentiment visualize.

**Comment by holdenhinkle** [comment]
Here's a page on GitHub that displays some metrics for successful CI workflows - https://github.com/department-of-veterans-affairs/vets-website/actions/workflows/continuous-integration.yml?query=is%3Asuccess++

As you can see, times vary greatly. I discussed this with Curt and we agree on the following:

- times vary because of test selection
- because of test selection, engineers will likely experience very different results
- an average of the time would be mostly meaningless (we can't think of how that metric might be useful)
- engineers probably aren't blocked when waiting for the CI workflow to run when their actively working on a branch. Perhaps they're blocked a little bit towards the end of working on a branch if checks are failing and they're having to fix things, but that's an interactive process where you wait and see, cancel the job, make changes and rerun the workflow, etc..

**Comment by pjhill** [comment]
This is done! We will establish a process and documentation to support this visualization under another ticket.
